src/App.js

```diff
@@ -21,48 +27,102 @@ export default function App() {
 }
 
 const Home = () => (
-  <div>
+  <div className="holder">
     <h1>🏀</h1>
     <h1 className="t">Miami Tech Runs</h1>
+    <img src="mtr-squad.jpg" alt="Miami Tech Runs Squad" className="squad" />
     <div className="p">
-      Bringing together the nation's emerging and established Founders, VCs, and
-      Creatives to build community, hoops, and mindfulness
+      Bringing together the Miami's emerging and established Founders, VCs, and
+      Creatives to build community, hoops, and mindfulness. Open membership!
     </div>
     <div className="m10">
       <Link to="/join">Click here for instructions to join</Link>
     </div>
     <div className="m10">
-      <Button
-        variant="primary"
-        size="lg"
-        onClick={() => {
-          window.location.href =
-            "https://telegram.me/collablandbot?start=VFBDI1RFTCNDT01NIy0xMDAxMDc0NzE2NTUx";
-        }}
-      >
-        Click here to get access to telegram
-      </Button>{" "}
+      <Link to="/tokens">
+        <Button variant="primary" size="lg">
+          Click here to buy $MTRS1 tokens
+        </Button>
+      </Link>
+    </div>
+    <div className="m10">
+      <p>
+        Next token holder game: November 13th -{" "}
+        <Link to="/games">full schedule.</Link>
+      </p>
+    </div>
+    <div>
+      <a href="https://www.instagram.com/miamitechruns/" target="_blank">
+        <img src="ig.png" className="ig" />
+      </a>
     </div>
   </div>
 );
 
-const Instructions = () => {
+const GameInstructions = () => {
+  return (
+    <div className="intructions">
+      <h1>Tentative upcoming games</h1>
+      <div>
+        <ul>
+          <li>
+            <strong>Next game:</strong> November 13th
```

Comment:
we should make this a array of objects and use `.map`  in the render

Reply:
Cool